src/Tooltip/tooltip-style.scss

```diff
@@ -46,6 +45,9 @@ $tooltip-distance: 4px;
                 vertical-align: top;
             }
         }
+        &.cursor {
```

Comment:
This class name is too generic, can you use `cursor-pointer` instead? Just to be more specific